### DISCUSSION TITLE
Are there any plans to support RabbitMQ management and deployment with Ansible ?

### DISCUSSION THREAD

**Discussion by bnachega** [rabbitmq/chef-cookbook#612] [General]
### Is your feature request related to a problem? Please describe.

Like we are using Chef recipes to mange and deploy RabbitMQ on Virtual Machines. Are there any plans to support or provide out of box scripts with Ansible playbooks or Automation ?

### Describe the solution you'd like

Support Rabbit MQ management using Ansible

**Comment by michaelklishin** [comment]
@bnachega RabbitMQ is open source software. The core team is very small and strongly prefers to spend time on RabbitMQ itself and its client libraries.

The Ansible community is welcome to develop a collection they need. In fact, a quick search suggests [there is one](https://github.com/ansible-collections/community.rabbitmq). I cannot vouch for its quality and whether it uses good deployment and [upgrade practices](https://rabbitmq.com/upgrade.html) but it's certainly a starting point.

The best source of good practices is Team RabbitMQ's set of [Kubernetes Operators](https://rabbitmq.com/kubernetes/operator/operator-overview.html), not this cookbook, by the way.
